content/millennium/r4/general-clinical/condition.md

```diff
@@ -178,8 +178,10 @@ Update an existing condition.
 
 _Implementation Notes_
 
-* Currently `problem-list-item` is supported.
+* Currently `problem-list-item` and `encounter-diagnosis` are supported.
 * Any field which is missing will be interpreted as nulling out or removing data from the resource. See [FHIR<sup>®</sup> Update] for additional details about update operations.
+* `abatementDateTime` is not supported for Conditions with a category of `encounter-diagnosis`.
+* `note` cannot be added if note already exists. An existing note field cannot be modified.
```

Comment:
I prefer adding them at the field level.

Reply:
Removed: https://github.com/cerner/fhir.cerner.com/pull/365/commits/2eb7b4a52698b32cdb3c8d3f9d0af6a11cbafb22#diff-c91ea4243fc73689536dc746c972a3d6L183